src/debugging-master/dashboard/styles.css

```
@@ -186,6 +186,24 @@ body {
+}
+
+.entry-row[data-expandable="true"] .dbg-log-wrap,
+.entry-row .dbg-json-inline-full,
+.entry-row .json-tree.select-text {
+    user-select: text;
+}
+
+.dbg-json-inline-full {
+    white-space: pre-wrap;
+    overflow-wrap: anywhere;
+    word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)